stage0/gc.go

```diff
@@ -125,9 +189,20 @@ func ForceMountGC(path, uuid string) error {
 		return fmt.Errorf("error getting mounts for pod %s from mountinfo: %v", uuid, err)
 	}
 
+	for i := len(mnts) - 1; i >= 0; i -= 1 {
+		mnt := mnts[i]
+		if mnt.mode != "-" && strings.Contains(mnt.mode, "shared") {
```

Comment:
https://github.com/torvalds/linux/blob/master/fs/proc_namespace.c#L154
```
	/* Tagged fields ("foo:X" or "bar") */
```

At the moment, the only tagged fields existing in the kernel are:
- `shared:%i`
- `master:%i`
- `propagate_from:%i`
- `unbindable`

So `strings.Contains(..., "shared")` would work. But it is not an elegant parsing.